### ISSUE TITLE
Undefined method `-' for nil:NilClass

### ISSUE THREAD

**Issue by mashkovtsevlx** [Shopify/theme-check#452]
I got a bug when was starting theme-check on my theme. A few days ago everything was working fine.

```
Exception while running 'LiquidTag#on_tag':

NoMethodError: undefined method '-' for nil:NilClass
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/node.rb:129:in 'inside_liquid_tag?'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/checks/liquid_tag.rb:16:in 'on_tag'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/checks.rb:42:in 'block in call_check_method'
  /usr/local/Cellar/ruby/3.0.2/lib/ruby/3.0.0/timeout.rb:97:in 'block in timeout'
  /usr/local/Cellar/ruby/3.0.2/lib/ruby/3.0.0/timeout.rb:35:in 'block in catch'
  /usr/local/Cellar/ruby/3.0.2/lib/ruby/3.0.0/timeout.rb:35:in 'catch'
  /usr/local/Cellar/ruby/3.0.2/lib/ruby/3.0.0/timeout.rb:35:in 'catch'
  /usr/local/Cellar/ruby/3.0.2/lib/ruby/3.0.0/timeout.rb:112:in 'timeout'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/checks.rb:41:in 'call_check_method'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/checks.rb:11:in 'block in call'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/checks.rb:10:in 'each'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/checks.rb:10:in 'call'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:35:in 'call_checks'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:22:in 'visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'block in visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'each'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'block in visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'each'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'block in visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'each'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'block in visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'each'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'block in visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'each'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'block in visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'each'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'block in visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'each'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'block in visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'each'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:24:in 'visit'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/visitor.rb:12:in 'visit_template'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/analyzer.rb:39:in 'block (2 levels) in analyze_theme'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/analyzer.rb:38:in 'each'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/analyzer.rb:38:in 'block in analyze_theme'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check.rb:57:in 'with_liquid_c_disabled'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/analyzer.rb:37:in 'analyze_theme'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/language_server/handler.rb:144:in 'block in analyze_and_send_offenses'
  /usr/local/Cellar/ruby/3.0.2/lib/ruby/3.0.0/benchmark.rb:293:in 'measure'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/language_server/handler.rb:143:in 'analyze_and_send_offenses'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/language_server/handler.rb:61:in 'on_text_document_did_open'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/language_server/server.rb:105:in 'process_request'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/language_server/server.rb:45:in 'block in listen'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/language_server/server.rb:44:in 'loop'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/language_server/server.rb:44:in 'listen'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/theme-check-1.5.2/lib/theme_check/language_server.rb:27:in 'start'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/lib/project_types/theme/commands/language_server.rb:8:in 'call'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/lib/shopify-cli/sub_command.rb:13:in 'call'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/lib/shopify-cli/command.rb:19:in 'call'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/vendor/deps/cli-kit/lib/cli/kit/executor.rb:16:in 'block (2 levels) in call'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/vendor/deps/cli-kit/lib/cli/kit/executor.rb:37:in 'block (2 levels) in with_logging'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/vendor/deps/cli-ui/lib/cli/ui/stdout_router.rb:169:in 'with_id'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/vendor/deps/cli-kit/lib/cli/kit/executor.rb:36:in 'block in with_logging'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/vendor/deps/cli-ui/lib/cli/ui.rb:176:in 'log_output_to'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/vendor/deps/cli-kit/lib/cli/kit/executor.rb:35:in 'with_logging'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/vendor/deps/cli-kit/lib/cli/kit/executor.rb:15:in 'block in call'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/vendor/deps/cli-kit/lib/cli/kit/executor.rb:45:in 'block (2 levels) in with_traps'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/vendor/deps/cli-kit/lib/cli/kit/executor.rb:55:in 'twrap'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/vendor/deps/cli-kit/lib/cli/kit/executor.rb:44:in 'block in with_traps'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/vendor/deps/cli-kit/lib/cli/kit/executor.rb:55:in 'twrap'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/vendor/deps/cli-kit/lib/cli/kit/executor.rb:43:in 'with_traps'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/vendor/deps/cli-kit/lib/cli/kit/executor.rb:14:in 'call'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/lib/shopify-cli/core/executor.rb:15:in 'call'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/lib/shopify-cli/core/entry_point.rb:24:in 'block in call'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/lib/shopify-cli/core/monorail.rb:20:in 'log'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/lib/shopify-cli/core/entry_point.rb:23:in 'call'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/bin/shopify:33:in 'block in <top (required)>'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/vendor/deps/cli-kit/lib/cli/kit/error_handler.rb:75:in 'handle_abort'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/vendor/deps/cli-kit/lib/cli/kit/error_handler.rb:21:in 'call'
  /usr/local/Cellar/shopify-cli/2.3.0/gems/shopify-cli-2.3.0/bin/shopify:32:in '<top (required)>'
  /usr/local/bin/shopify:6:in 'load'
  /usr/local/bin/shopify:6:in '<main>'
'''

Template: 'snippets/locksmith-content-variables.liquid'
Node: 'Liquid::Comment'
Markup:
'''
comment 
'''
Check options: '{:min_consecutive_statements=>5}'
Theme Check Version: 1.5.2
Ruby Version: 3.0.2
Platform: x86_64-darwin20
Muffin mode: activated

------------------------
```

**Comment by mashkovtsevlx** [comment]
@charlespwd Just sent. Thank you.

**Comment by charlespwd** [comment]
Fixed in 1.6.1 (just released), you can reinstall `shopify-cli` to get the updates before we release a new version of Shopify CLI.

```
brew uninstall shopify-cli
brew install shopify-cli
```
